### ISSUE TITLE
Normalization should not be done in the wrapper

### ISSUE THREAD

**Issue by DavidGOrtega** [nmslib/hnswlib#453]
Normalization should be done on the C++ side (if any, my vectors come out from my models already normalised).
If not every time the C++ code is used normalization has to be reimplemented

**Comment by yurymalkov** [comment]
Hi @DavidGOrtega,
Sure, we can move it. PRs are welcome.
Thanks!

**Comment by kiplingw** [comment]
I agree. I think it would be a good idea to have this functionality implemented internally, as opposed to being binding specific.
